presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -269,6 +272,18 @@ and a file system location of ``s3://test_bucket/test_schema/test_table``:
         location = 's3://test_bucket/test_schema/test_table')
     )
 
+Session Properties
+-------------------
+
+Session properties set behaviour changes for queries executed within the given session.
+
+============================================= ======================================================================
+Property Name                                 Description
+============================================= ======================================================================
+``iceberg.delete_as_join_pushdown_enabled``   Overrides the behaviour of the connector property
```

Comment:
I didn't set the default since it's an override for the connector property and the default is documented there, do you think it should be here as well?

Reply:
Yes, I think so. Here's my reasoning: 

* The session property (iceberg.delete_as_join_pushdown_enabled) is a separate property from the configuration property (iceberg.delete-as-join-pushdown-enabled), so I would like the doc for the session property to be self-contained. 

* As it is written now, the reader must infer that the values to set the session property to are the same as the configuration property - but it isn't stated that that is the case here. They'll probably figure it out, but why make them? Alternatively, could add "The values for `iceberg.delete_as_join_pushdown_enabled` are the same as `iceberg.delete-as-join-pushdown-enabled` ", but including the Default value for the session property _with_ the session property is shorter, less ambiguous, and doesn't require the reader to go find `iceberg.delete-as-join-pushdown-enabled` in the other table.

That was a lot of words but I wanted to explain why, and an alternative I'd considered. Does this make sense?